docs/admin/upgrading_to_1.7.0.md

```diff
@@ -0,0 +1,23 @@
+# Upgrading to 1.7.0
+
+This note describes the necessary steps to upgrade to Stellio 1.7.0
+
+## Contact entity service
+
+Stellio now allows to send periodic notifications, for this it's necessary to add new variables to contact the entity service in the docker-compose. 
+In subscription-service, please add the following variables: 
+
+```
+# Client registration used to get entities from entity-service
+spring.security.oauth2.client.registration.keycloak.authorization-grant-type=client_credentials
+spring.security.oauth2.client.registration.keycloak.client-id=client-id
+spring.security.oauth2.client.registration.keycloak.client-secret=client-secret
+spring.security.oauth2.client.provider.keycloak.token-uri=https://my.sso/token
+application.entity.service-url=http://localhost:8082
+```
```

Comment:
not totally clear since you are talking about docker compose but you describe raw properties, not env vars used by docker compose.

maybe split it into 2 parts:
- we added four properties: ...
- if using Stellio in docker compose, here is how you can customize them: ...